camp/src/main/java/ch/bfh/zodnrk/camp/service/impl/DefaultHeroService.java

```diff
@@ -1,17 +1,20 @@
 package ch.bfh.zodnrk.camp.service.impl;
 
 import ch.bfh.zodnrk.camp.model.Hero;
+import ch.bfh.zodnrk.camp.repository.HeroRepository;
 import ch.bfh.zodnrk.camp.service.HeroService;
+import org.springframework.beans.factory.annotation.Autowired;
 
 import javax.inject.Named;
 import java.util.Random;
 
 @Named
 public class DefaultHeroService implements HeroService {
 
-    private final Random rng = new Random();
-
     private static final double DEFAULT_HERO_HEALTH = 100.0;
+    private final Random rng = new Random();
+    @Autowired
```

Comment:
We should not mix Spring & JSR330 annotations:

- Spring: `@Service` & `@Autowired`
- JSR330: `@Named` & `@Inject`

I suggest going with the JSR330 standard

Reply:
That's a good point.
We started with using [constructor Injection](https://github.com/zodnrk/project-heroes/blob/master/camp/src/main/java/ch/bfh/zodnrk/camp/service/conf/ServiceConfig.java#L18) for better testability.
We should either continue that, or switch to field injection everywhere.

Reply:
Ah, yeah, your totally right. I'd suggest we continue with constructor injection. Thanks for pointing this out.